docs/developer/tools/lib-alert-tree/examples/example.py

```diff
@@ -0,0 +1,44 @@
+from lib_alert_tree.models import DerivedAlert as Derived
+from lib_alert_tree.models import ExistingAlert as Existing
+from lib_alert_tree.models import Relationship
+from lib_alert_tree.models import severity_pair
+from lib_alert_tree.cli import generate_cli
+
+# Existing alerts are defined with their name and labels only.
+FOO_WARNING = Existing("FooServiceTooManyErrors", severity="warning")
+FOO_CRITICAL = Existing("FooServiceTooManyErrors", severity="critical")
+
+# There are shortcuts for defining the severity.
+BAR_BAZ_WARNING = Existing.warning("BarAlmostOutOfSpace", bar="baz")
+BAR_BAZ_CRITICAL = Existing.critical("BarAlmostOutOfSpace", bar="baz")
+
+# Derived alerts are built from a list of children and a relationship type (the default
```

Comment:
How is it implicit? I feel it to be simpler to understand when you link all children from a parent, rather than linking the parent to all children individually. What would you change?

Or do you mean the default being set to "ANY"?

Reply:
yes the ANY part

Reply:
OK I'll set it back to being a mandatory argument.

Reply:
Done in 542b283.